store/controller.py

```diff
@@ -37,36 +42,113 @@ def __get_target(self, target_cls):
+                                            self.remove_product_from_store(
+                                                target)
+                                        case 'add':
+                                            self.add_product_to_store(target)
+                        case Product.__name__:
+                            self.change_product_price(target, target_type)
         else:
-            if target_type == 'STORE':
+            if target_type.__name__ == Store.__name__:
                 self.create_store()
-            elif target_type == 'PRODUCT':
+            elif target_type.__name__ == Product.__name__:
                 self.create_product()
-        self.sync()
-
-    def get_store(self):
-        pass
+        self.sync_stores()
+        self.sync_products()
 
     def create_store(self):
-        pass
-
-    def update_store(self):
-        pass
+        address = input('Enter the address of the store: ')
+        while True:
+            capacity = input('Enter the capacity of the store: ')
+            if capacity.isdigit() and int(capacity) > 0:
+                break
+        self.stores.append(Store(address, capacity))
 
-    def delete_store(self):
-        pass
+    def update_store(self, store, store_class):
+        if not store_class.get_by_id(store.id):
+            return
+        while True:
+            change = input(
+                'Enter the info which you want to update [address, capacity]: ').upper()
+            if change in ("ADDRESS", "CAPACITY"):
+                if change == 'ADDRESS':
+                    store.address = input(
+                        "Enter a new address for the store: ")
+                else:
+                    while True:
+                        capacity = input('Enter a new capacity of the store: ')
+                        if capacity.isdigit():
+                            store.capacity = int(capacity)
+                            break
+                break
 
-    def get_product(self):
-        pass
+    def delete_store(self, store):
+        for i, el in enumerate(self.stores):
+            if el.id == store.id:
+                del self.stores[i]
 
     def create_product(self):
-        pass
+        name = input('Enter the name of the product: ')
+        category = input('Enter the category of the product: ')
+        while True:
+            price = input('Enter the price of the product: ')
+            if price.isdigit() and int(price) > 0:
+                break
+        self.products.append(Product(name, price, category))
+
+    def add_product_to_store(self, store):
+        product_id = input("ENter the ID of the product you want to add: ")
+        quantity = input(
+            "Now, enter the quantity of the product you want to add: ")
+        store.add_product(product_id, quantity)
+
+    def remove_product_from_store(self, store):
+        product_id = input("Enter the ID of the product you want to add: ")
```

Comment:
Suggestion: Checking if product with the entered `ID` exists or not would be a good approach